src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -69,7 +69,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     <AssemblySearchPaths>{CandidateAssemblyFiles};{HintPathFromItem};{TargetFrameworkDirectory};{RawFileName}</AssemblySearchPaths>
     <DesignTimeAssemblySearchPaths>$(AssemblySearchPaths)</DesignTimeAssemblySearchPaths>
   </PropertyGroup>
+  
+  <!-- Default glob settings -->
+  <PropertyGroup>
+    <GlobalExclude>$(GlobalExclude);bin\**;obj\**;packages\**</GlobalExclude>
+  </PropertyGroup>
 
+  <ItemGroup Condition=" '$(OverrideDefaultGlobs)' != 'True' ">
+    <Compile Include="**\*.cs" Exclude="$(GlobalExclude);wwwroot\**" Condition=" '$(OverrideDefaultCompileGlob)' != 'True' " />
+    <EmbeddedResource Include="**\*.resx" Exclude="$(GlobalExclude);wwwroot\**" Condition=" '$(OverrideDefaultEmbeddedResourceGlob)' != 'True' " />
+  </ItemGroup>
```

Comment:
:+1: for me, and ok for F#, if i can use `OverrideDefaultCompileGlob` to disable **\*.cs include.

It should be always possibile to individually disable implicit behaviour with a single feature flag, so dont remove the condition because if you use `$(DefaultLanguageSourceExtension)` instead that's gonna break F# badly.